spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/servlet/error/ErrorMvcAutoConfiguration.java

```diff
@@ -253,13 +253,13 @@ public void render(Map<String, ?> model, HttpServletRequest request,
 
 		private String getMessage(Map<String, ?> model) {
 			Object path = model.get("path");
-			String message = "Cannot render error page for request [" + path + "]";
+			StringBuilder message = new StringBuilder(String.format("Cannot render error page for request [%s]", path));
```

Comment:
Why introduce here a `String.format` call when the spirit of this PR is to improve performance?